include/deal.II/base/table.h

```diff
@@ -1903,7 +1903,7 @@ TableBase<N,T>::reinit (const TableIndices<N> &new_sizes,
   // if new size is nonzero: if necessary allocate additional memory, and if
   // not fast resize, zero out all values)
   values.resize_fast (new_size);
-  if (!fast)
+  if (!omit_zeroing_entries)
```

Comment:
Therefore, I think the name of the argument is actually fine for the simple case. We might add a comment at the generic 'reinit' method that for non-trivial classes the memory will be initialized in any case.